src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+$mainAccent:#060b35;
+$secondary: #616070;
+$blueAccent: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainAccent;
+    margin: 40px 0 0;
+  }
+
+  &__productCode {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+    margin: 4px 0 0;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 0;
+  }
+
+  &__stars {
```

Comment:
if you are familiar with scss imports it'd be nice to move stars BEM block to an own file (`stars.scss`) the same as card BEM block (`card.scss`). Plus create separated file for variables (`variables.scss`). And then just import three files to `index.scss`